Frameworks/GLKit/GLKViewController.mm

```diff
@@ -91,13 +101,12 @@ - (void)viewWillDisappear:(BOOL)animated {
 }
 
 - (BOOL)_renderFrame {
-    [_calendar setToNow];
-    WFDateTime* dt = [_calendar getDateTime];
-    int64_t frameTime = dt.universalTime;
+    _calendar.SetToNow();
+    WF::DateTime frameTime = _calendar.GetDateTime();
 
-    _timeSinceFirstResume = static_cast<NSTimeInterval>(frameTime - _firstStart) * 0.0000001;
-    _timeSinceLastResume = static_cast<NSTimeInterval>(frameTime - _lastStart) * 0.0000001;
-    _timeSinceLastUpdate = static_cast<NSTimeInterval>(frameTime - _lastFrame) * 0.0000001;
+    _timeSinceFirstResume = timeSpanToInterval(frameTime - _firstStart);
+    _timeSinceLastResume = timeSpanToInterval(frameTime - _lastStart);
+    _timeSinceLastUpdate = timeSpanToInterval(frameTime - _lastFrame);
```

Comment:
Indeed. Since we're using the calendar from `Windows.Globalization`, it should be primed with the user's timezone and location; we're not, as far as I can tell, compensating for distance from UTC or anything.